presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -440,6 +440,11 @@ Convenience Extraction Functions
     Returns the ISO day of the week from ``x``.
     The value ranges from ``1`` (Monday) to ``7`` (Sunday).
 
+.. function:: day_of_week(x, y) -> bigint
+
+        Returns the day of the week from ``x`` based on the start day of the week ``y``.
```

Comment:
What are x and y in terms of type? X is date? or timestamp? what about y?

Reply:
Why does this not say 

day_of_week(x)

Returns the day of the week for the specified date. The values range....

Reply:
X is date or timestamp or timestampz and Y is integer ranging from 1 to 7. 

Not sure of the day_of_week(x) because this was already existing function and I almost used the same definition for my new function, since it is just a variant of it.

Reply:
This needs to be added to the docs ...